src/Model/Context.php

```diff
@@ -98,11 +117,11 @@ public function has($name)
     }
 
     /**
-     * @return array
+     * @return array|\stdClass
      */
     public function jsonSerialize()
     {
-        return $this->data;
+        return $this->data ?: new \stdClass;
```

Comment:
@hfinck yes you right it can be usefull, but it's should not return a `stdClass`, return `null` will be better ? What do you think ?